apps/emacs/emacs.talon

```diff
@@ -0,0 +1,343 @@
+app: emacs
+-
+tag(): user.tabs
+tag(): user.splits
+tag(): user.line_commands
+
+# ----- GENERAL ----- #
+#suplex: key(ctrl-x)
+cancel: user.emacs("keyboard-quit")
+exchange: user.emacs("exchange-point-and-mark")
+execute: user.emacs_meta_x()
+execute {user.emacs_command}$: user.emacs(emacs_command)
+execute <user.text>$:
+    user.emacs_meta_x()
+    user.insert_formatted(text, "DASH_SEPARATED")
+evaluate | (evaluate | eval) (exper | expression): user.emacs("eval-expression")
+prefix: user.emacs("universal-argument")
+prefix <user.number_signed_small>: user.emacs_prefix(number_signed_small)
+
+abort recursive [edit]: user.emacs("abort-recursive-edit")
+browse kill ring: user.emacs("browse-kill-ring")
+fill paragraph: user.emacs("fill-paragraph")
+insert char: user.emacs("insert-char")
+occurs: user.emacs("occur")
+other scroll [down]: user.emacs("scroll-other-window")
+other scroll up: user.emacs("scroll-other-window-down")
+package autoremove: user.emacs("package-autoremove")
+package list | [package] list packages: user.emacs("list-packages")
+reverse (lines | region): user.emacs("reverse-region")
+save buffers kill emacs: user.emacs("save-buffers-kill-emacs")
+save some buffers: user.emacs("save-some-buffers")
+sort lines: user.emacs("sort-lines")
+sort words: user.emacs("sort-words")
+
+go directory: user.emacs("dired-jump")
+other go directory: user.emacs("dired-jump-other-window")
+
+[toggle] debug on error: user.emacs("toggle-debug-on-error")
+[toggle] input method: user.emacs("toggle-input-method")
+[toggle] truncate lines: user.emacs("toggle-truncate-lines")
+[toggle] word wrap: user.emacs("toggle-word-wrap")
+
+manual: user.emacs("man")
+manual <user.text>:
+    user.emacs("man")
+    user.insert_formatted(text, "DASH_SEPARATED")
+
+# BUFFER SWITCHING #
+# many things override these keybindings, so we don't use user.emacs
```

Comment:
Should remove this comment since we do actually use user.emacs here now — and add switch-to-buffer and switch-to-buffer-other-window to emacs_commands.csv?

Reply:
done